services/media/service/.env.template

```diff
@@ -13,6 +13,10 @@ DATABASE_LOGIN=media_service_login
 DATABASE_LOGIN_PASSWORD=media_service_login_pwd
 DATABASE_GQL_ROLE=media_service_gql_role
 
+# NOTE: If these values change the old database replication slots must be explicitly dropped.
+TRANSACTIONAL_OUTBOX_REPLICATION_SLOT=ott_media_service_transactional_outbox_slot
+TRANSACTIONAL_INBOX_REPLICATION_SLOT=ott_media_service_transactional_inbox_slot
```

Comment:
Because it's not clear which PR will be completed first - this, or
`feature/localization` - this PR might still require some changes that are
already in the `feature/localization`
- Root .env.template - add `POSTGRESQL_REPLICATION_PORT`
- adjust `infra/docker-compose.yml` to include replication parameters